.buildkite/pipeline.yml

```
@@ -0,0 +1,10 @@
+steps:
+  - block: "Publish branch to GitHub registry"
+    branches: "!master"
```

Should this just be on master branch?
```suggestion
    branches: "master"
```